deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -8,8 +8,6 @@ spec:
     kind: UnifiedPushServer
     listKind: UnifiedPushServerList
     plural: unifiedpushservers
-    shortNames:
```

Comment:
This should not be removed. What version of the `operator-sdk` do you have? You may need to update to a non-released version.

Reply:
0.7.0

Reply:
Could you try the latest (0.8.1 I think), and if this is still getting removed, installing from master will fix that.